app/Resources/views/security/login.html.twig

```diff
@@ -0,0 +1,79 @@
+    </header>
+    <section id="content" role="main">
+        <header class="aui-page-header">
+            <div class="aui-page-header-inner">
+                <div class="aui-page-header-main">
+                    <h1>{% block pageHeader %}Atlassian Vendor Tools{% endblock %}</h1>
+                </div>
+            </div>
+        </header>
+        <div class="aui-page-panel" style="min-height: 400px">
+            <div class="aui-page-panel-inner">
+                <section class="aui-page-panel-content">
+                    {% block body %}
+                        <div class="avt-login-wrap">
+                            {% if error %}
+                                <div>{{ error.messageKey|trans(error.messageData, 'security') }}</div>
+                            {% endif %}
+
+                            <form action="{{ path("login") }}" method="post" class="aui">
+
+                                <div class="field-group">
+                                    <label for="username">Username<span class="aui-icon icon-required">(required)</span></label>
+                                    <input type="text" id="username" name="_username" value="{{ last_username }}"
+                                           required="required"
+                                           class="text medium-field"/>
+                                </div>
+
+                                <div class="field-group">
+                                    <label for="password">Password<span class="aui-icon icon-required">(required)</span></label>
+                                    <input type="password" id="password" name="_password" required="required"
+                                           class="text medium-field"/>
+                                </div>
+                                <fieldset class="group">
+                                    <div class="checkbox">
+                                        <label for="remember_me"><input type="checkbox" id="remember_me" name="_remember_me" class="checkbox"
+                                                                        value="on"/>Stay logged in</label>
+                                    </div>
+                                </fieldset>
+                                <div class="buttons-container">
+                                    <input class="button submit" type="submit" value="Login">
+                                </div>
+                            </form>
+                        </div>
+                    {% endblock %}
+                </section>
+            </div>
+        </div>
+    </section>
+    <footer id="footer" role="contentinfo">
+    </footer>
+</div>
+<script src="{{ asset('assets/jquery/dist/jquery.min.js') }}"></script>
+<script src="{{ asset('assets/aui/js/aui.min.js') }}"></script>
+<script src="{{ asset('assets/vendor/highcharts.js') }}"></script>
+{#<script src="{{ asset('assets/aui/js/aui-experimental.min.js') }}"></script>#}
```

Comment:
Please remove commented block

Reply:
Sorry. I've left it in my view blueprint and've forgotten to remove before push. Fixed.